runserver.sh

```diff
@@ -10,11 +10,15 @@
 set -e
 
 standalone() {
+  local DJANGO_HTTP_PORT=${DJANGO_HTTP_PORT:-8000}
```

Comment:
idk if this kind of intentional shadowing is kosher in bash, but I'm OK with it

Reply:
Looks good to me. You can do all sorts of fun stuff with variables like this in bash. Another neat trick is to fail the script if an expected environment variable isn't defined:

```bash
#!/usr/bin/env bash
: "${DJANGO_HTTP_PORT?This message will print and the script will exit if not defined}"
```